bool cam_shoot
bool cam_zoom_in_full
bool cam_zoom_out_full
bool cam_zoom_in
bool cam_zoom_out
uint8 cam_change_view

bool select_cam_1
bool select_cam_2
int32 sci_cam_select